### ISSUE TITLE
validator_name="Error: Usage information not found"

### ISSUE THREAD

**Issue by dan-kazenoff-vcc** [tedder/miner_exporter#14]
Certain metrics output an error for the "validator_name" parameter. Examples:

system_usage{job="Helium_Validator", resource_type="CPU", validator_name="Error: Usage information not found for the given command RPC to 'val_miner@127.0.0.1' failed: {'EXIT', {{case_clause, {error, {no_matching_spec,["info","name"]}}}, [{blockchain_console,command,1, [{file,"blockchain_console.erl"}, {line,16}]}, {rpc,'-handle_call_call/6-fun-0-',5, [{file,"rpc.erl"},{line,197}]}]}}"}

system_usage{job="Helium_Validator", resource_type="Memory", validator_name="Exact ERTS version (10.7.1) match not found, instead using 10.7.1. The release may fail to run."}

Typically, the metric is duplicated and the last "system_usage" in the group correctly displays the validator_name, but the metrics above it display the error.

Other metrics where the same error pops up:
validator_block_age, validator_container_uptime, validator_version_info.

See attached image:

![image](https://user-images.githubusercontent.com/70960809/120679382-194bb680-c467-11eb-920e-4e15fa2f7d16.png)

**Comment by tedder** [comment]
Yeah, it doesn't handle errors well and it should. Waiting for the new API update unless you/someone wants to do some quick regex style validation on the validator name.
